types/relay-runtime/lib/store/RelayStoreTypes.d.ts

```diff
@@ -457,6 +457,35 @@ type LogEvent =
           name: 'execute.unsubscribe';
           transactionID: number;
       }>
+    | Readonly<{
+        name: 'network.info',
+        transactionID: number,
+        info: any,
```

Comment:
The TS `unknown` type semantically maps best to [Flow’s `mixed`](https://flow.org/en/docs/types/mixed/)

```suggestion
        info: unknown,
```